api-specificatie/openapi.yaml

```diff
@@ -385,6 +391,8 @@ components:
     Adres:
       type: object
       properties:
+        nummeraanduidingIdentificatie:
+          $ref: 'https://raw.githubusercontent.com/lvbag/BAG-API/master/Technische%20specificatie/Yaml%27s/20191209-1-lvbag-individuele-bevragingen-20180601/lvbag/imbag/v20180601/nen3610.yaml#/NEN3610IDNum'
         straatnaam:
           title: openbareruimte naam
           description: De verkorte versie van een naam die aan een openbare ruimte is toegekend in een daartoe strekkend formeel gemeentelijk besluit.
```

Comment:
Als dit de verkorte straatnaam is, dan mag deze slechts 24 karakters lang zijn. Een beter voorbeeld van een verkorte naam is dan denk: de langste straatnaam van Nederland: 'Laan van de landinrichtingscommissie Duiven-Westervoort', de verkorte naam daarvan is:  'Ln vd l D Westervoort'. Moeten niet zowel de verkorte als de volledige straatnaam in Adres worden teruggegeven?

Reply:
Dat is er 1 voor de productowner.s Ik maak een issue aan.